XP-Pen_Artist16_Pro/index.md

```diff
@@ -0,0 +1,19 @@
+---
+VID: "28bd"
+PID: "900a"
+vendor: XP-Pen
+product: Artist16 Pro
+image: Artist16_Pro
```

Comment:
Please use the full tablet identifier here (as for images for every other tablet). We need to name the image files in full (not just the model name), because in some cases we have data on a tablet from one manufacturer, but the photo from a different brand, and we need to record which model is exactly on the photo.